src/components/Users.js

```diff
@@ -0,0 +1,76 @@
+import { useEffect, useState } from "react";
+import { BaseUrl } from "../Constants";
+import axios from "axios";
+import localStorageService from "../LocalStorageService";
+import Table from "react-bootstrap/Table";
+import { Spinner } from "react-bootstrap";
+
+export default function Users() {
+  const [users, setUsers] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    const token = localStorageService.get("auth-token");
+    let config = {
+      method: "get",
+      maxBodyLength: Infinity,
+      url: BaseUrl + "chat/users/",
+      headers: {
+        Authorization: `token ${token}`,
+      },
+    };
+
+    axios
+      .request(config)
+      .then((response) => {
+        setIsLoading(false);
+        setUsers(response.data);
+      })
+      .catch((error) => {
+        console.log(error);
+      });
+  }, []);
+
+  return (
+    <div className="container">
+       <div class="row justify-content-between align-items-center mt-4 mb-4">
+            <div class="col-auto">
+                <h2>Users</h2>
+            </div>
+        </div>
+      <Table striped bordered hover>
+        <thead>
+          <tr>
+            <th>Id</th>
+            <th>Name</th>
+            <th>Email</th>
+          </tr>
+        </thead>
+        <tbody>
+          {!isLoading && (
```

Comment:
Here, we only want to show loading spinner if the data is being loaded which means if the loading flag is true otherwise we have to show the list of users. so here we need to remove this check `{isLoading && (`